src/Doctrine/Annotation/Tokens.php

```diff
@@ -58,7 +58,7 @@ public static function createFromDocComment(PhpToken $input, array $ignoredTags
             $index = 0;
             $nbScannedTokensToUse = 0;
             $nbScopes = 0;
-            while (null !== $token = $lexer->peek()) {
+            while ($token = $lexer->peek()) {
```

Comment:
Should not be done. Because `doctrine/annotations` introduced a breaking change for that.
Question is if the check should be to call `empty()` or something.

Reply:
Maybe check for both `null` and `[]` ? But perhaps the change introduced in `doctrine/annotations` was not intentional. Maybe wait to see how they respond.